//
// Created by wj on 18-10-10.
//

#ifndef LOGINSERVER_MATCHINGHANDLER_H
#define LOGINSERVER_MATCHINGHANDLER_H


#include "BaseHandler.h"

class matchingHandler: BaseHandler {


public:
    matchingHandler(const CoonPrt coonPrt, protocol_ &aProtocol);
private:
    //开始匹配
    static map<int,
    void playmatching();

};


#endif //LOGINSERVER_MATCHINGHANDLER_H
